engine/gamesys/src/dmsdk/script.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 The Defold Foundation
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#ifndef DMSDK_SCRIPT_H
+#define DMSDK_SCRIPT_H
+
+#include <dmsdk/script/script.h>
+#include <dmsdk/gamesys/script.h>
+
+#endif // DMSDK_SCRIPT_H
```

Comment:
Created this new overarching script.h that references the difference implementations.